cnd/src/network/mod.rs

```diff
@@ -128,6 +128,36 @@ impl Swarm {
             local_peer_id,
         })
     }
+
+    /// Takes all the data we know about a 'created' swap.  Executes the
+    /// appropriate communication protocols in order to send data to the peer
+    /// and receive data from the peer (remaining swap parameters) in order to
+    /// finalize this swap.
+    pub fn finalize_created_swap<A, B>(
+        &mut self,
+        swap_id: SwapId,
+        swap_digest: SwapDigest,
+        secret_hash: SecretHash,
+        alpha_identity: A,
```

Comment:
What are we going to need these for?

Reply:
This is the identities sent using the identity_protocl i.e., bitcoin_identity, ethereum_identity, lightning_identity.